fake-cucumber/javascript/src/FakeStepDefinitions.ts

```diff
@@ -0,0 +1,103 @@
+import { messages } from 'cucumber-messages'
+import {
+  ParameterTypeRegistry
+} from 'cucumber-expressions'
+import uuidv4 from 'uuid/v4'
+
+import Argument from 'cucumber-expressions/dist/src/Argument'
+import CucumberExpression from 'cucumber-expressions/dist/src/CucumberExpression'
```

Comment:
Same as above

Reply:
the thing for this one is that, when I use `import { CucumberExpression } from `cucumber-expression`, I can not use `CucumberExpression` as a type for the variables, so I have to write weird stuff like `public expression: any = new CucumberExpression` (which is pretty ugly too)